packages/grid/_modules_/grid/GridComponent.tsx

```diff
@@ -52,6 +52,7 @@ import { useLocaleText } from './hooks/features/localeText/useLocaleText';
 import { useGridCsvExport } from './hooks/features/export';
 import { useGridInfiniteLoader } from './hooks/features/infiniteLoader';
 import { visibleGridRowCountSelector } from './hooks/features/filter/gridFilterSelector';
+import { classnames } from './utils/classnames';
```

Comment:
To rebase, this utils was replaced with the canonical API of the codebase.

Reply:
Updated